### PR TITLE
Remove meeting link in contributing doc

### PR DESCRIPTION
**Pull request by ThomsonTan**
## Changes

We updated Zoom meeting in README in https://github.com/open-telemetry/opentelemetry-cpp/pull/887, but we also have the same link in contributing doc which needs to be fixed as well. Instead, this PR removes the duplicated link in the contributing doc.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Unit tests have been added
* [ ] Changes in public API reviewed

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -141,8 +141,8 @@ on each other), the owner should try to get people aligned by:
   split it up.
 
 If none of the above worked and the PR has been stuck for more than 2 weeks, the
-owner should bring it to the [OpenTelemetry C++ SIG
-meeting](https://zoom.us/j/8203130519). The meeting passcode is _77777_.
+owner should bring it to the OpenTelemetry C++ SIG meeting. See
+[README.md](README.md#Contributing) for the meeting link.
```
Comment (reyang) [inline CONTRIBUTING.md:145]: `README.md#Contributing` or `README.md#contributing` or doesn't matter (uppercase or lowercase)?

Comment (ThomsonTan) [reply CONTRIBUTING.md:145]: Seems casing doesn't matter in URL. [this](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/README.md#CoNtrIbutinG) URL works as well in my browser.

Comment (reyang) [reply CONTRIBUTING.md]: I don't think so.

https://www.w3.org/TR/WD-html40-970708/htmlweb.html

_URLs in general are case-sensitive (with the exception of machine names). There may be URLs, or parts of URLs, where case doesn't matter, but identifying these may not be easy. **Users should always consider that URLs are case-sensitive.**_

**Review by reyang — APPROVED** [review]
LGTM.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ThomsonTan) [reply CONTRIBUTING.md]: Thanks for the sharing. It makes sense for the URL to be case-sensitive as it shows up in many short URLs. Perhaps github does the toleration.

**Pull request merged**